plugins/woocommerce/lib/composer.json

```diff
@@ -9,7 +9,8 @@
 	"require-dev": {
 		"mobiledetect/mobiledetectlib": "^3.74",
 		"psr/container": "^1.1",
-		"pelago/emogrifier": "7.3.0"
+		"pelago/emogrifier": "7.3.0",
+		"league/iso3166": "^4.3.3"
```

Comment:
I am not familiar with how dependencies are managed here. Do we need this dependency here, since we are adding the source under `lib/packages/League`?

Reply:
That's how the dependency is aliased (see the [README.md](https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/lib/README.md#adding-packages) from this directory):
```
In order to avoid including the original dependencies in the root autoloader we must utilize require-dev for them. Composer treats require dependencies as transitive while require-dev dependencies get ignored by consumers.

- Add package to require-dev section of composer.json
- Add package slug to extra/mozart/packages section of composer.json
- Run composer run-script build-lib from the root directory (You should now see the package in packages/VendorName/PackageName or classes)
```